### PR TITLE
docs(response-rewrite): change image source from jsdelivr to github

### LINKED ISSUE #7149: help request: can not load image of the plugin response-rewrite important tips for `ngx.exit`
### Description

[response-rewrite](https://apisix.apache.org/docs/apisix/plugins/response-rewrite/) important tips image for `nginx.exit` load failed. 

Note:
* My PC couldn't connect to [image server](https://cdn.jsdelivr.net/gh/Miss-you/img/picgo/20201113010623.png).
sendpix0

https://github.com/apache/apisix/blob/aefffe914f9f3ec187d3db6688c02d22785cf400/docs/en/latest/plugins/response-rewrite.md?plain=1#L129

Suggest turn it into Markdown Table.

### Environment

- APISIX version (run `apisix version`):
- Operating system (run `uname -a`):
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

### PR DESCRIPTION
**Pull request by kwanhur**
### Description

Change plugin response-rewrite `ngx.exit` image source. Bcz jsdelivr cann't be accessed in somewhere, so load it from github.

Fixes #7149 

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/response-rewrite.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ X-Server-balancer_addr: 127.0.0.1:80
 
 如果你在 `access` 阶段执行了 `ngx.exit`，该操作只是中断了请求处理阶段，响应阶段仍然会处理。如果你配置了 `response-rewrite` 插件，它会强制覆盖你的响应信息（如响应代码）。
 
-![ngx.edit tabular overview](https://cdn.jsdelivr.net/gh/Miss-you/img/picgo/20201113010623.png)
+![ngx.exit tabular overview](https://user-images.githubusercontent.com/2106987/170849580-aa296d3d-b43a-404a-8c88-720afb2b7c08.png)
```
Comment (hf400159) [inline docs/zh/latest/plugins/response-rewrite.md:128]: This picture is a table, can we turn it into a table? Like the table under properties.

Comment (juzhiyuan) [reply docs/zh/latest/plugins/response-rewrite.md:128]: Good idea!

Comment (juzhiyuan) [reply docs/zh/latest/plugins/response-rewrite.md:128]: This way is more miantainable.

Comment (kwanhur) [reply docs/zh/latest/plugins/response-rewrite.md:128]: Ok, I'll change with plaintext table.

Comment (juzhiyuan) [reply docs/zh/latest/plugins/response-rewrite.md]: Thanks! cc @hf400159 to have a check 🙏

**Review by hf400159 — APPROVED** [review]
LGTM

**Review by tzssangglass — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Pull request merged**